book/02-git-basics/sections/viewing-history.asc

```diff
@@ -264,7 +265,7 @@ In <<limit_options>> we'll list these and a few other common options for your re
 [cols="2,4",options="header"]
 |================================
 | Option                | Description
-| `-(n)`                | Show only the last n commits
+| `-(n)`                | Show only the last n commits (really a shortcut for `-n (n)`, so `-5` and `-n 5` are equivalent)
```

Comment:
I worry a bit about the line length in the table cell here. Some renderers might do a terrible job of wrapping an overlong line. Can we change it to this?

```
| `-(n)` or `-n (n)`    | Show only the last n commits
```